saleor/cart/__init__.py

```diff
@@ -52,6 +51,8 @@ def __str__(self):
 
     @classmethod
     def for_session_cart(cls, session_cart, discounts=None):
+        from saleor.product.models import Product
```

Comment:
To avoid a circular import (you should never import models in `__init__` as it can be read before the app registry is ready).